view/frontend/web/js/pagseguro.js

```diff
@@ -78,6 +78,18 @@ RMPagSeguro.prototype.addCardFieldsObserver = function(obj){
         /*jQuery(cpf).keyup(function( event ) {
             obj.updateCreditCardToken();
         });*/
+        jQuery(ccExpYrVisibileElm).keyup(function( event ) {
+
+            if(jQuery(ccExpYrVisibileElm).length == 0) {
+                ccExpYr = '200' + jQuery(ccExpYrVisibileElm).val();
```

Comment:
If lenght == 0, why I would do 200 + val, knowing val is empty?
Did I get this wrong?